oasislmf/pytools/gul/common.py

```diff
@@ -68,3 +68,10 @@
                                              ('loss', oasis_float),
                                              ]))
 gulSampleslevelRec_size = gulSampleslevelRec.size
+
+
+Keys = {'LocID': np.int32,
```

Comment:
`Keys` is already defined in  `pytools/getmodel/common.py` and it should not be redefined here.
We should import `Keys` from that module, or viceversa, but not defining it twice.

Reply:
Good point about duplication, it seems like we should add a file like `oasislmf/pytools/common.py` for dictionaries/data common to all of the pytool components (rather than sharing them across sub-dirs)

Reply:
Indeed, Sam, this is indeed what I've started doing here: https://github.com/OasisLMF/OasisLMF/pull/1118